### PR TITLE
Simplified presentation

### PR DESCRIPTION
**Pull request by kingofthebongo2008**
Typos are fixed
Direct X 12 text is reduced, added some colors
Added several diagrams to the destiny parallel architectures

### CODE REVIEW

File: lectures/11.rendering_2.md [hunk 1]
```diff
@@ -320,9 +321,10 @@ GPU Stalls
...
-* Issue data request for item N, block
-* Issue data request for item N + 1, block
-* Issue data request for item N + 2, block
+
+* Issue data request for item N, block <!-- .element class="fragment" data-fragment-index="0" -->
+* Issue data request for item N + 1, block <!-- .element class="fragment" data-fragment-index="1" -->
+* Issue data request for item N + 2, block <!-- .element class="fragment" data-fragment-index="2" -->
 
 * Data comes for item N, execute
 * Data comes for item N + 1, execute
```
Comment (nikoladimitroff) [inline lectures/11.rendering_2.md]: you animated the "Issue data request" but not the "data comes part"; I guess you meant to do both

File: lectures/11.rendering_2.md [hunk 2]
```diff
@@ -592,36 +591,30 @@ DirectX 12 Descriptor Heaps
...
-* Render Target View ( for output from the shaders)
-* Depth Stencil View (for the depth buffer).
-* Sampler ( this is a special hardware that performs sampling )
-* From the theory of signals
-* Sampling means to read the values of the signals
-* Geometry is a signal, Textures are signal.
+* <span style="color: green">Shader Resource View</span> (for reading from shaders)
+* <span style="color: green">Render Target View</span> ( for output from the shaders)
+* <span style="color: green">Depth Stencil View</span> (for the depth buffer).
+* <span style="color: green">Sampler ( this is a special hardware that performs sampling )
```
Comment (nikoladimitroff) [inline lectures/11.rendering_2.md]: you are missing the closing < / span>

File: lectures/11.rendering_2.md [hunk 3]
```diff
@@ -638,102 +631,56 @@ DirectX 12 Work Generation
...
  * Can block a gpu queue to wait on another gpu queue
  * Can block the cpu until the gpu passes a point
-* Gpu
+* <span style="color: green">Gpu</span>
  * queue->Signal(fence, 100);
  * anotherQueue->Wait(fence, 100);
-* Cpu
- * m_fence->SetEventOnCompletion(fence, 100); WaitForSingleObject(fence, INFINITE);
+* <span style="color: red">Cpu</span>
+ * fence->SetEventOnCompletion(fence, 100); WaitForSingleObject(fence, INFINITE);
```
Comment (nikoladimitroff) [inline lectures/11.rendering_2.md]: Use single backticks  for inline code -  `i++`

File: lectures/11.rendering_2.md [hunk 4]
```diff
@@ -950,28 +881,27 @@ RenderPass::RenderPass(FrameGraphBuilder& builder)
...
+  RenderGraphMutableResource renderTarget)
 {
   // Declare resource dependencies
   m_input = builder.read(input, readFlags);
   m_renderTarget = builder.write(renderTarget, writeFlags);
 }
 ```
 --- VERTICAL SLIDE ---
-Frame Graph resource example
+c
```
Comment (nikoladimitroff) [inline lectures/11.rendering_2.md:907]: typo I guess?

**Review by nikoladimitroff — APPROVED** [review]

File: lectures/11.rendering_2.md [hunk 5]
```diff
@@ -548,36 +550,33 @@ Now when one allocates the heap he must manage the memory inside. He/she must an
 * How about multiple threads allocations?
 * When I free something, does the gpu use it?
 * What about the fragmentation of this memory?
-
 * What about if the OS requires eviction?
-* Alignment of resources
+* Alignment of resources ex.
```
Comment (Alekssasho) [inline lectures/11.rendering_2.md]: ex ?

**Review by Alekssasho — APPROVED** [review]

**Comment by nikoladimitroff** [comment]
Merged in 03d60bf0ed1821f02283cf0517913ea78c284915

**Pull request closed without merging**